### ISSUE TITLE
[Request For Comment] The boost optimize for Key frame is amazing, but can be push to upstream?

### ISSUE THREAD

**Issue by ghost** [LianjiaTech/keframe#11]
The boost optimize for Key frame is amazing, but can be push to upstream?
Any thought push the optimize to flutter/flutter upstream?
I guess coding is easy ,but let it compatiable with all flutter version is hard .

**Comment by ghost** [comment]
See also: https://github.com/flutter/flutter/issues/96256
ping @Nayuta403 
Would you please interest in keep touch with Flutter team?

**Comment by Nayuta403** [comment]
Yeah, no problem. I have also applied for the Flutter Favorite Project.
